### ISSUE TITLE
Magento 1.9

### ISSUE THREAD

**Issue by GordonLesti** [GordonLesti/Lesti_Smtp#3]
Should I make Lesti_Smtp Magento 1.9 ready?

**Comment by razbakov** [comment]
yes

**Comment by razbakov** [comment]
Have you seen this module? https://github.com/aschroder/Magento-SMTP-Pro-Email-Extension

**Comment by GordonLesti** [comment]
@razbakov yes I know it. I guess it's better maintained and already works for 1.9. The question for me is, does anybody need Lesti_Smtp?

**Comment by razbakov** [comment]
I switched to aschroder/smtp_pro as it has more features: 
- test from backend
- show errors
